wolox-books/src/scss/application.scss

```diff
@@ -0,0 +1,5 @@
+@import '~wolox-equalizer/equalizer';
+@import url('https://fonts.googleapis.com/css?family=Roboto:400,700');
```

Comment:
you can leave it like this: `@import 'https://fonts.googleapis.com/css?family=Roboto:400,700';`